js/chart.js

```diff
@@ -0,0 +1,64 @@
+import html from './html.js';
+
+function makeTemplate() {
+    return html`
+        <canvas id="myChart" width="400" height="400"></canvas>
```

Comment:
```suggestion
<div>
        <canvas id="myChart" width="400" height="400"></canvas>
</div>
```
The canvas element can be a bit tricky, it needs to be wrapped in a container element in order to render to the screen properly.